modules/sortmerna.nf

```diff
@@ -6,7 +6,7 @@
 process sortmerna {
     label 'sortmerna'
 
-    publishDir "${params.output}/${params.sortmerna_dir}", mode: 'copy', pattern: "${name}.other.fastq"
+    publishDir "${params.output}/${params.sortmerna_dir}", mode: 'copy', pattern: "${name}*.other.fastq"
```

Comment:
simply the sortmerna output by using paired-end data was not published because the pattern did not match